### DISCUSSION TITLE
Module size too big

### DISCUSSION THREAD

**Discussion by ROCKEIRO67** [sibiraj-s/ngx-editor#529] [Q&A] [answered]
Hi guys, I implemented this rich text editor in my app. However the module in my app that imports the NgxEditorModule gets to a whooping 1.03MB in size. I used npx webpack-bundle-analyzer dist/stats.json command and saw that ngx-editor and it's dependicies from prosemirror made it so big. Is there a way to only import certain components such as the ngxeditorcomponent and menucomponent aswell as Editor and Toolbar classes?
![image](https://github.com/sibiraj-s/ngx-editor/assets/22966540/fb334728-912a-4d49-817b-096eb66c5f44)

**Comment by sibiraj-s** [comment, marked as answer]
Hi. Yes, as you mentioned the size is because of prosemirror, only the ones required to make the editor are included. I am afraid even though you just import the editor you will get all.

**Comment by hiepxanh** [comment]
you should lazy import it
